packages/atlas/src/components/_navigation/SidenavBase/SidenavBase.styles.ts

```diff
@@ -189,30 +100,29 @@ export const LegalLinksWrapper = styled.span`
 
 export const LegalLink = styled(Link)`
   text-decoration: none;
-  font: ${cVar('typographyDesktopT200Strong')};
-  letter-spacing: ${cVar('typographyDesktopT200StrongLetterSpacing')};
-  text-transform: ${cVar('typographyDesktopT200StrongTextTransform')};
+  color: ${cVar('colorTextMuted')};
 
   &:hover {
-    color: ${oldColors.gray[400]};
+    color: ${cVar('colorText')};
   }
 
   &:focus {
-    color: ${oldColors.gray[400]};
-  }
-
-  &:active {
-    color: ${oldColors.gray[500]};
+    color: ${cVar('colorText')};
   }
 `
 
 export const StyledHamburgerButton = styled(HamburgerButton)`
   position: fixed;
+  margin-left: ${sizes(3)};
   z-index: ${zIndex.sideNav};
-  left: ${sizes(3)};
-  top: ${sizes(2)};
-
+  margin-top: ${sizes(2)};
   ${media.md} {
-    top: ${sizes(4)};
+    margin-top: ${sizes(4)};
```

Comment:
You haven't replied why we're using margins instead of top/left now. Check the other comment thread